source/api/presentation/3.0/index.md

```diff
@@ -722,16 +722,17 @@ The value _MUST_ be an array of JSON objects. Each item _MUST_ have the `id` and
 
 #### 3.4.2. Internal Links
 
-##### within
-A link to another resource that contains this resource, such as a Manifest that is part of a Collection. When encountering the `within` property and the referenced resource is not included in the current representation, clients might retrieve the referenced resource to contribute to the processing of this resource. For example, if a Canvas is encountered as a stand alone resource, it might be `within` a Manifest that includes other contextual information, or a Manifest might be `within` a Collection, which would aid in navigation.
+##### partOf
+
+A link to another resource that contains this resource, such as a Manifest that is part of a Collection. When encountering the `partOf` property and the referenced resource is not included in the current representation, clients might retrieve the referenced resource to contribute to the processing of this resource. For example, if a Canvas is encountered as a stand alone resource, it might be part of a Manifest that includes other contextual information, or a Manifest might be part of a Collection, which would aid in navigation.
```

Comment:
I think the first two sentences are awkward, I suggest starting: "A link to another resource that contains this resource, for example a link from a Manifest to a Collection that it is part of. When a client encounters the `partOf` property, it might retrieve the referenced resource if it is not included in the current representation, in order to contribute to the processing of this resource."

Reply:
:+1: